editor.planx.uk/src/pages/Team.tsx

```diff
@@ -128,13 +128,15 @@ export function AddButton({
 
 interface FlowItemProps {
   flow: any;
+  flows: any;
```

Comment:
Use of the ``any`` type followed current practices in the ``Team`` component, may be worth a refactor in future to follow TS best practices